adaptive_breakpoints/lib/src/adaptive_breakpoints.dart

```diff
@@ -0,0 +1,377 @@
+import 'package:flutter/material.dart';
+
+/// Adaptive Window in Material has five different window sizes. Each one can be
+/// broken into smaller bits.
+///
+/// Extra small represents phones and small tablets on portrait view.
+/// Small represents tablets in portrait view and phones on landscape view.
+/// Medium is going to represent large tablets in landscape view.
+/// Large represents computer screens.
+/// Extra large represents TVs.
+enum AdaptiveWindow {
+  xs,
+  s,
+  m,
+  l,
+  xl,
+}
+
+extension AdaptiveWindowExtension on AdaptiveWindow {
```

Comment:
It feels like if this were defined in terms of range values, as below, it'd be simpler to understand

Reply:
Good idea, I changed all the variables to range values.